app/experimenter/nimbus-ui/src/services/hooks.test.tsx

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Maybe we should rename `services/` to `lib/`?

FWIW, I had something like [this](https://dev.to/farazamiruddin/an-opinionated-guide-to-react-folder-structure-file-naming-1l7i) or [this](https://www.robinwieruch.de/react-folder-structure) in mind when I started the `services/` directory. So, I would put this file in `hooks/index.tsx` and keep `services/` just for external network or DOM API code.

But, this is a road leading straight to a bikeshed and IMO not worth the energy to draft an ADR. So it's probably worth simplifying.

Reply:
That's a good point. I'm happy to adopt the structure from that first link, or move it to `lib/`. What do you think @LZoog?

Either way I would want to do this in a followup so I can ignore the ref locally

Reply:
Yeah when I reviewed this I almost suggested moving it into `lib/` or another directory since I didn't really consider hooks "services," but was OK with it if you preferred them in `services/`. I'm down for moving it out.